File: Programming Basics/JavaScript/Conditional Statements Advanced/Labs/fruit-or-vegetable.js
function solve(input) {
    const product = input[0];

    if (product === "banana" || product === "apple" || product === "kiwi" || product === "cherry" || product === "lemon" || product === "grapes") console.log("fruit");
    else if (product === "tomato" || product === "cucumber" || product === "pepper" || product === "carrot") console.log("vegetable");
    else console.log("unknown");
}

File: Programming Basics/C++/Conditional Statements Advanced/Labs/FruitOrVegetable/FruitOrVegetable.cpp
#include <iostream>
using namespace std;

int main()
{
	string product;
	cin >> product;

	if (product == "banana" || product == "apple" || product == "kiwi" || product == "cherry" || product == "lemon" || product == "grapes") cout << "fruit" << endl;
	else if (product == "tomato" || product == "cucumber" || product == "pepper" || product == "carrot") cout << "vegetable" << endl;
	else cout << "unknown" << endl;
}
